common/tools/configtxgen/encoder/encoder_test.go

```diff
@@ -190,9 +190,19 @@ var _ = Describe("Encoder", func() {
 		It("translates the config into a config group", func() {
 			cg, err := encoder.NewOrdererGroup(conf)
 			Expect(err).NotTo(HaveOccurred())
-			Expect(len(cg.Policies)).To(Equal(2)) // BlockValidation automatically added
+			Expect(len(cg.Policies)).To(Equal(1))
+			expectedImplicitMetaPolicy := &cb.ImplicitMetaPolicy{
+				SubPolicy: "Admins",
+				Rule:      cb.ImplicitMetaPolicy_ANY,
+			}
+			value, err := proto.Marshal(expectedImplicitMetaPolicy)
+			Expect(err).NotTo(HaveOccurred())
+			expectedAdminPolicy := &cb.Policy{
+				Type:  int32(cb.Policy_IMPLICIT_META),
+				Value: value,
+			}
 			Expect(cg.Policies["SamplePolicy"]).NotTo(BeNil())
-			Expect(cg.Policies["BlockValidation"]).NotTo(BeNil())
+			Expect(cg.Policies["SamplePolicy"].Policy).To(Equal(expectedAdminPolicy))
```

Comment:
Can you convert this inline with `protoutil.MarshalOrPanic` and remove lines 194-203?
```
Expect(cg.Policies["SamplePolicy"].Policy).To(Equal(&cb.Policy{
				Type: int32(cb.Policy_IMPLICIT_META),
				Value: protoutil.MarshalOrPanic(&cb.ImplicitMetaPolicy{
					SubPolicy: "Admins",
					Rule:      cb.ImplicitMetaPolicy_ANY,
				}),
			}))
```

Reply:
Done